tests/device_tests/test_msg_signtx.py

```diff
@@ -908,7 +992,6 @@ def test_two_changes(self, client):
                     proto.ButtonRequest(code=B.ConfirmOutput),
                     request_output(1),
                     request_output(2),
-                    proto.ButtonRequest(code=B.ConfirmOutput),
```

Comment:
Note here that we no longer make any prompts when there are only two change outputs.

Reply:
why? is there a common usecase for this?

Reply:
A simple thing you can use this for is creating a change-output which has the same value as the spend-output, so that an observer can't tell which one belongs to the recipient and which one is yours. In addition to that one you also have a second classic change-output.

This can of course be generalized to having _n_ change-outputs with the same value as the spend-output, but then it gets complicated. Like you should avoid spending two of the equal-valued change-outputs together in another transaction, because then an observer can deduce that those were not the payment outputs. So 2 satisfies the basic use-case.